### ISSUE TITLE
Appointment scheduling within Simplero

### ISSUE THREAD

**Issue by stephenhutson** [Simplero/Roadmap#365]
I would love to Simplero to have a module for appointment scheduling. The goal would be to replace other apps like Calendly or Acuity Scheduling. It would need:

- A beautiful landing page with all event types
- Unique links for the events home page and customizable unique links per event type
- A simple UI for booking appointments
- Integration with calendar apps and automatic detection of busy times
- Can accept payment at the time of booking
- Can trigger other automations/events within Simplero

This would be so nice to have right within the software instead of paying for another app.

**Comment by Ryan-Henderson** [comment]
I agree. We currently use Calendly. Having this automatically link to our SImplero account would be awesome.

**Comment by mortenspindler** [comment]
And... Booking for physical classes/courses as well,
and the ability to check in when coming to a class or a trainer to set a student as no-show,
so we can keep track of payment and how many people really attend classes (yoga)
